src/services/statement.of.capital.service.ts

```diff
@@ -0,0 +1,17 @@
+import { createPrivateOAuthApiClient } from "./api.service";
+import { ConfirmationStatementService, StatementOfCapital } from "private-api-sdk-node/dist/services/confirmation-statement";
+import { Session } from "@companieshouse/node-session-handler";
+import Resource, { ApiErrorResponse } from "@companieshouse/api-sdk-node/dist/services/resource";
+
+export const getStatementOfCapitalData = async (session: Session, companyNumber: string): Promise<StatementOfCapital> => {
+  const client = createPrivateOAuthApiClient(session);
+  const csService: ConfirmationStatementService = client.confirmationStatementService;
+  const response: Resource<StatementOfCapital> | ApiErrorResponse = await csService.getStatementOfCapital(companyNumber);
+  const status = response.httpStatusCode as number;
+  if (status >= 404) {
```

Comment:
Should be 400 not 404?
Else its going to treat 400, and 401 etc as a success response and try cast it to a statement of capital